src/components/adaptive-cards/TextBlock/TextBlock.scss

```diff
@@ -0,0 +1,8 @@
+$C: #{$WEBEX_COMPONENTS_CLASS_PREFIX}-adaptive-cards-text-block;
+
+.#{$C} {
+  display: -webkit-box;
```

Comment:
let's not add these properties unless we really need them (maxLines is defined). Otherwise all text blocks will have overflow:hidden and display:webkit-box, which might not be an issue but who knows.

so please add an optional class name .has-max-lines and put this definition under it.